api/AudioData.json

```diff
@@ -538,6 +538,55 @@
             "deprecated": false
           }
         }
+      },
+      "timestamp": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/AudioData/timestamp",
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/AudioData/timestamp#examples is wrong, if you'd like to fix some docs :)